### DISCUSSION TITLE
[BluetoothLE] ACCESS_BACKGROUND_LOCATION

### DISCUSSION THREAD

**Discussion by Avots** [shinyorg/shiny#790] [Q&A] [answered]
After upgrading from version 2.0.0.2542 preview to 2.3.0.2906 Google rejected the app because wants ACCES_BACKGROUND_LOCATION permission.
In the manifest we do not add this permission, only indicated by Shiny:
"android.permission.ACCESS_COARSE_LOCATION"
"android.permission.ACCESS_FINE_LOCATION"
"android.permission.BLUETOOTH"
"android.permission.BLUETOOTH_ADMIN"
"android.permission.BLUETOOTH_PRIVILEGED"

There anything that can be done?

I tried but it didn't work
uses-permission android: name = "android.permission.ACCESS_BACKGROUND_LOCATION" tools: node = "remove"

Regards

**Comment by aritchie** [comment, marked as answer]
You can fork the project and comment out the attribute where it is being inserted.  It will likely be removed in the next version.

**Reply by aritchie** [reply]
I've decided to include this as part of the next preview release which should drop in a few days

**Comment by Avots** [comment]
Thanks for the feedback!
I'm just using Shiny.BluetoothLE.
Waiting for the next preview! :)

**Comment by Avots** [comment]
Sorry if I posted here, I don't know if it's allowed.
I updated to version 2.3.0.2936-preview to be able to test the removal of ACCESS_BACKGROUND_LOCATION and it happens.
Once connected, I search for bluetooth and return the connected status. If searching a second time, it returns connected, but the status is disconnected.

![image](https://user-images.githubusercontent.com/79453107/140510577-951d1097-9b97-44a4-b06b-311a6e247587.png)

**Reply by aritchie** [reply]
Don't hijack the issue from one topic to the next.  If you have an issue, follow the full issue template and include a reproducible sample.
